containers/DvmContracts.ts

```diff
@@ -0,0 +1,62 @@
+import { createContainer } from "unstated-next";
```

Comment:
This whole file is shown as a diff because I re-named it. I chose to add the store state into the `DVMContracts.ts` rather than creating a separate store contract state container.

The main addition is that the `Promise.all` returns the `Store` address in addition to the `Oracle` address.

Reply:
Makes sense to group Store and Voting into the same file